saleor/graphql/page/types.py

```diff
@@ -245,35 +305,41 @@ def wrap_with_channel_context(
             and requestor.has_perm(PagePermissions.MANAGE_PAGES)
         ):
             return (
-                SelectedAttributesAllByPageIdLoader(info.context)
+                AssignedAttributesAllByPageIdLoader(info.context)
                 .load(page.id)
                 .then(wrap_with_channel_context)
             )
         return (
-            SelectedAttributesVisibleInStorefrontPageIdLoader(info.context)
+            AssignedAttributesVisibleInStorefrontPageIdLoader(info.context)
             .load(page.id)
             .then(wrap_with_channel_context)
         )
 
-    @staticmethod
+    @classmethod
     def resolve_attribute(
-        root: ChannelContext[models.Page], info: ResolveInfo, slug: str
+        cls, root: ChannelContext[models.Page], info: ResolveInfo, slug: str
+    ):
+        return cls._resolve_assigned_attribute(root, info, slug)
```

Comment:
How about moving `cls._resolve_assigned_attribute` to `resolvers.py` like in other cases?